ros2doctor/ros2doctor/api/network.py

```diff
@@ -66,6 +66,11 @@ def check(self):
             return result
 
         has_loopback, has_non_loopback, has_multicast = _check_network_config_helper(ifcfg_ifaces)
+        if not has_loopback and not has_non_loopback:
+            # no flags found, otherwise one of them should be True.
+            result.add_warning('No flags found. \
```

Comment:
I'm not sure if it should be a warning in the result since there's nothing a user can do to resolve it. Maybe this should be printed, or maybe `result` should have an even softer category like `result.add_note()`?

Reply:
I think this scenario is rare (or I would like to keep it rare) enough to use a print statement instead of extending the result category, but good point and I will modify it.